.circleci/cimodel/data/pytorch_build_definitions.py

```diff
@@ -158,8 +184,19 @@ def gen_dependent_configs(xenial_parent_config):
 def gen_docs_configs(xenial_parent_config):
     configs = []
 
-    for x in ["pytorch_python_doc_push", "pytorch_cpp_doc_push", "pytorch_doc_test"]:
-        configs.append(HiddenConf(x, parent_build=xenial_parent_config))
+    configs.append(
+        HiddenConf(
+            "pytorch_python_doc_build",
+            parent_build=xenial_parent_config
+        )
+    )
+    configs.append(
+        DocPushConf(
+            "pytorch_python_doc_push",
+            parent_build="pytorch_python_doc_build",
+            branch="site-v1.6.0",
+        )
+    )
```

Comment:
Just confirming -- is this what does the actual push to `site-v1.6.0`?

Reply:
e.g., when we release 1.7.0 documentation, do we have to add this line to that PR?